ehr2meds/data/generation/helpers.py

```
@@ -58,9 +64,24 @@ def greater_than_datetime(min_date, end=2020):
 
-
 # Specialized functions for DST dataset
 def honuge(start_year, end_year):
     week = random.randint(1, 52)
     year = str(random.randint(start_year, end_year))[-2:]
     return f"{year}{week:02d}"
+
+# Function to mix multiple functions
+def mix_function(functions, probabilities):  
+    if not np.isclose(sum(probabilities), 1.0):
+        raise ValueError(f"Probabilities must sum to 1.0, got {sum(probabilities)}")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)